bootstrap/pkg/apis/apps/configconverters/converters.go

```diff
@@ -123,7 +125,24 @@ func LoadConfigFromURI(configFile string) (*kfconfig.KfConfig, error) {
 				strings.Join(versions, ", "), apiVersionSeparated[1]),
 		}
 	}
-	return converter.ToKfConfig(appDir, configFileBytes)
+	cwd, err := os.Getwd()
+	if err != nil {
+		return nil, &kfapis.KfError{
+			Code:    int(kfapis.INTERNAL_ERROR),
+			Message: fmt.Sprintf("could not get current directory for KfDef %v", err),
+		}
+	}
+
+	return converter.ToKfConfig(cwd, configFileBytes)
+}
+
+func isCwdEmpty() string {
```

Comment:
where is this func used?

Reply:
removed. I think it's moved to somewhere else.